lib/flying_penguin/duffel/client.ex

```diff
@@ -55,6 +35,41 @@ defmodule FlyingPenguin.Duffel.Client do
     end
   end
 
```

Comment:
On line 17:
I think this could be cleaned up a little bit to make it more readable. Something like this:

```

url = "https://api.duffel.com/air/offer_requests?return_offers=true"
  case HTTPoison.post(url, request_body(search_params), request_headers, [recv_timeout: 20000]) do
    {:ok, %Response{status_code: 201, body: body }} ->
      raw
      |> :zlib.gunzip
      |> Poison.decode(keys: :atoms)

    {:ok, %Response{status_code: 404}} ->
      IO.puts "Not found :("

    {:error, %Error{reason: reason}} ->
      IO.inspect "there was an error"
      IO.inspect reason
  end